### PR TITLE
Ethicalpage

### PR DESCRIPTION
**Pull request by RachellCalhoun**
i made an ethical and responsibility page, view, model and fixed up responsive css.

### CODE REVIEW

**Comment by RachellCalhoun** [comment]
Can you review this? @jinpark I'm trying to get caught up. This looks fine on my end.

File: content/models.py [hunk 1]
```diff
@@ -50,3 +50,14 @@ class EduProgram(models.Model):
 
 	def __str__(self):
 		return self.title
+
+class EthicalPost(models.Model):
+	title = models.CharField(blank=True, null=True, max_length=100, help_text="Only enter a title if order is 1. This is the page title.")
+	img = models.ImageField(upload_to='aboutus',blank=True, null=True, help_text="Upload image corresponding to Text")
+	text = models.TextField(blank=True, null=True, help_text="Enter a short description.")
+	order = models.CharField(max_length=2, help_text="Enter a number. 1 will be at the top of the page")
```
Comment (jinpark) [inline content/models.py]: why charfield? maybe you want integerfield?

File: content/models.py [hunk 2]
```diff
@@ -50,3 +50,14 @@ class EduProgram(models.Model):
...
+
+
+
+	def __str__(self):
+		return self.order
```
Comment (jinpark) [inline content/models.py]: you probably want either self.title or self.text here

Comment (RachellCalhoun) [inline content/models.py]: self.text here would be a paragraph and looks really messy. I used charfield for order so I could use it for the name. (integerfield won't work)
But I'll just have them enter a title (even if the users don't see it, it'll still be useful for the admin page), and use self.title. That way I can change the charfield to integer

**Comment by RachellCalhoun** [comment]
Fixed :100: ?

**Comment by hassanabidpk** [comment]
I am getting an error when I visit `http://127.0.0.1:8000/`

**Comment by RachellCalhoun** [comment]
what error? Mine is ok /ethics is the page name. There isn't a homepage right now.

**Comment by hassanabidpk** [comment]
Okay thats why.  Model and everything else looks okay to me. But design (css) requires lots of fixes

**Pull request merged**